ddd-koans/EduTools and basics/Task Description/task.html

```diff
@@ -0,0 +1,34 @@
+<html>
+<h1>Task description</h1>
+<p>
+    Task description provides all information that is needed to solve the problem.
+    Please read it carefully not to miss any important requirement for the task to be solved.
+</p>
+<p>
+    The task description bar also provides button to <i>Check</i>, <i>Run</i> and go to <i>Next</i> task/lesson buttons.
+</p>
```

Comment:
it would be nice to explain how to move to the previous/next task using arrows on the top right corner - there was a question about it.